app/app.py

```diff
@@ -33,6 +33,24 @@
 
 BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 
+# Prompt Templates
+SIMPLE_TEMPLATE = "Context: {context}\n\nQuestion: {question}\n\n"
+
+COMPLEX_TEMPLATE = """Use the following pieces of context to answer the question at the end.
```

Comment:
More a question but how precise do we need to be in our language here? For example, could the prompt be:

```
Use the following context to answer the question.
If you don't know the answer, say that you don't know and do not try to make up an answer.
Use a maximum of three sentences and keep the answer concise.

{context}

Question: {question}
Answer:
```